frontend/src/components/panels/NodeGraph.svelte

```diff
@@ -60,16 +73,35 @@
 		return sparse;
 	}
 
-	function buildNodeCategories(nodeTypes: FrontendNodeType[], searchTerm: string): [string, FrontendNodeType[]][] {
-		const categories = new Map();
+	type NodeCategoryDetails = {
+		nodes: FrontendNodeType[];
+		shouldBeOpen: boolean;
```

Comment:
Would simply calling this `open` be clearer?

Reply:
yes.